core/src/main/resources/common/message/OffsetCommitResponse.json

```diff
@@ -0,0 +1,52 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "name": "OffsetCommitResponse",
+  "validVersions": "0-3",
+  "fields": [
+    {
+      "name": "offset",
+      "versions": "0+",
+      "type": "int64"
+    },
+    {
+      "name": "metadata",
+      "versions": "0+",
+      "type": "string"
+    },
+    {
+      "name": "timestamp",
+      "versions": "0",
+      "type": "int64"
+    },
+    {
+      "name": "commitTimestamp",
+      "versions": "1+",
+      "type": "int64"
+    },
```

Comment:
I think that we can combine these two together. The field was only renamed in V1. Having `commitTimestamp` with versions `0+` and dropping `timestamp` entirely should be fine.

Reply:
good idea!